product_plm_import/models/plm_product_mapping.py

```diff
@@ -21,6 +21,13 @@ class PlmProductMapping(models.Model):
         .selection,
         required=True,
     )
+    purchase_description_rule = fields.Selection(
+        selection=[
+            ("standard", "Standard Description"),
+            ("plm", "PLM Description"),
```

Comment:
To be more descriptive.
```suggestion
            ("standard", "Standard: [Generic Name] / [Drawing] / [Spec]"),
            ("generic_name", "Generic Name),
```